### ISSUE TITLE
UI change

### ISSUE THREAD

**Issue by akshaykalola28** [akshaykalola28/Online-Electricity-Billing-System#3]
Need the contributor for UI changes.

**Comment by kranand6** [comment]
Hi Akshay, I am willing to work on this. Can this be assigned to me?
Thank you!

**Comment by akshaykalola28** [comment]
Yes. Off course. You can start work on this. Show your creativity. Good Luck.

**Comment by cchigo** [comment]
I would also love to work on this.

**Comment by akshaykalola28** [comment]
@cchigo Welcome to collaborate. Please show your excellency. It really appreciated.

**Comment by cchigo** [comment]
Hi, can you give the app a name? and do you have a logo for it? I would be working on the UI this weekend. Thank you.

**Comment by akshaykalola28** [comment]
This project is only for educational purpose. So, I didn't give a particular name to this project. But, if you want to create a logo then use the default project name (Online Electricity Billing System).

**Comment by cchigo** [comment]
Alright then

**Comment by unclejo** [comment]
can i join this project can help to upgrade programming structure to ModelView ViewModel(MVVM) architecture

**Comment by akshaykalola28** [comment]
@unclejo Yes. Of course.

**Comment by hariharan25-03** [comment]
i cannot able to  add bill  for customers showing error

**Comment by hariharan25-03** [comment]
please 
give some solution faster

**Comment by CodeStealerHAHA** [comment]
can u guys manage to log in?

**Comment by unclejo** [comment]
@akshaykalola28 can i start change the ui?

**Comment by akshaykalola28** [comment]
@CodeStealerHAHA @hariharan25-03 It's due to a firebase server error. Please fork this repo. and set up your firebase account in this project. 

@unclejo Yes. You can. Please make a separate branch for it and create PR after completion.

**Comment by unclejo** [comment]
> lease make a separate branch for it and create PR after completion.

but is it okay to change whole UI project and use module library to store all colors and styles in a single module?

**Comment by akshaykalola28** [comment]
Yes. You can use your own expertise.

**Comment by unclejo** [comment]
@akshaykalola28 i created a branch woud you like to visit?

**Comment by unclejo** [comment]
@akshaykalola28 i create pull request to be merge check it if you have available time.
